dashboard/app/helpers/levels_helper.rb

```diff
@@ -172,8 +172,8 @@ def app_options
 
     if @level.is_a? Blockly
       @app_options = blockly_options
-    elsif @level.is_a? DSLDefined
-      @app_options = dsl_defined_options
+    elsif @level.is_a?(DSLDefined) || @level.is_a?(FreeResponse)
```

Comment:
assessment as a keyword is used in a .script file and iirc any level can be one of them.  Being DSL defined is orthogonal to that, I think.